pajbot/tmi.py

```diff
@@ -1,8 +1,20 @@
 class TMI:
```

Comment:
you should introduce a new enum class (https://docs.python.org/3/library/enum.html) that has the three variants for whisper limits. Preferrably add the string parsing for the config value to the enum class (e.g. `def from_config_value(config_value)`)